crimsobot/utils/image.py

```diff
@@ -115,6 +116,12 @@ async def fetch_image(ctx: Context, arg: Optional[str]) -> Image.Image:
     session = aiohttp.ClientSession()
 
     async def open_img_from_url(url: str) -> Image.Image:
+        if 'tenor.com/view' in url:
```

Comment:
someone could possibly put tenor.com/view in a random other url but idk what they'd really achieve by that, just wanted to at least call it out

Reply:
"Congratulations, you made a Discord bot return an error! You are very intelligent."